server/api/entryRouters.ts

```diff
@@ -0,0 +1,46 @@
+import express, { Request, Response, NextFunction } from "express";
+import { Entry } from "../db/index.js";
+import { EntryAttributes } from "../db/models/Entry.model.js";
+const router = express.Router();
+
+router.get("/", async (req: Request, res: Response, next: NextFunction) : Promise<void>=> {
+  try {
+    const entries: EntryAttributes[] = await Entry.findAll();
```

Comment:
Under current this would give all entries from all accounts. This need to force check for isAdmin and authenticate the user.